File: Python/BMI_calculator.py
height = float(input("Please Enter your height in cm: "))
weight = float(input("Please Enter your weight in kg: "))

BMI = weight / (height/100)**2

print(f"You BMI is {BMI}")

if BMI <= 18.4:
    print("You are underweight. Do Something!!!")
elif BMI <= 24.9:
    print("You are healthy.")
elif BMI <= 29.9:
    print("You are over weight. Do Something!!! ")
elif BMI <= 34.9:
    print("You are severely over weight. Do Something!!! ")
elif BMI <= 39.9:
    print("You are obese. Do Something!!!")
else:
    print("You are severely obese. God Bless You!!!")

File: BMI calculator using JAVA/BMI_calculator.java
//Java program to calculate the BMI

//Importing the Scanner package
import java.util.Scanner;

//Main / Drived Class
public class Main
{
    //Main Function
    public static void main(String[] args) {
        //Creating the object of Scanner Class
        Scanner input = new Scanner(System.in);
        //Giving the hint to user what has to enter
        System.out.println("Enter your weight unit (kg or lbs): ");
        //Taking Weight Unit from user
        String unit = input.nextLine();
        //Giving the hint to user what has to enter
        System.out.println("Enter your Weight : ");
        //Taking weight from user
        double weight = input.nextDouble();

        double height = 0;
        //Checking which weight unit has been selected by user
        if(unit.equals("kg")){
            //If unit is kg
            //Giving the hint to user what has to enter
            System.out.println("Enter your Height(In Meters) : ");
            //Taking height(in Meters) from user
            height = input.nextDouble();
        }else if(unit.equals("lbs")){
            //Giving the hint to user what has to enter
            System.out.println("Enter your Height(In Inches) : ");
            //Taking height(in Inches) from user
            height = input.nextDouble();
        }
        //Declaring a double type parameter
        double bmi;
        /*Switch Cases on weight units because BMI Formula changes according to
        /* the Weight Units
        */
        switch (unit){
            //If unit is kg
            case "kg":
                //caluclate the bmi with formula and store it in variable
                bmi = weight / (height * height);
                //print the BMI
                System.out.println("YOUR BMI IS : "+ bmi +" kg/m2");
                break;
            //If unit is lbs
            case "lbs":
                //caluclate the bmi with formula and store it in variable
                bmi = 703 * (weight / (height * height));
                //print the BMI
                System.out.println("YOUR BMI IS : "+ bmi +" lbs/in2");
                break;
            default:
                System.out.println("Please choose correct weight unit");
        }
    }
} 
